ios/FluentUI/Command Bar/CommandBarTokens.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class MSFCommandBarTokens: MSFTokensBase, ObservableObject {
```

Comment:
This class does not need to be an ObservableObject as the CCB implementation is not SwiftUI.